src/agri_ai/langchain_tools/field_registration_tool.py

```
@@ -0,0 +1,252 @@
+                    "region": field_info["region"],
+                    "address": "詳細住所未設定"
+                },
+                "current_cultivation": None,
+                "cultivation_history": [],
+                "next_scheduled_work": None,
+                "irrigation_system": "不明",
+                "greenhouse_type": None,
+                "created_at": datetime.now(),
+                "updated_at": datetime.now(),
+                "status": "active",
+                "notes": f"ユーザー登録: LINEから追加された圃場"
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)